batcher/aligned-batcher/src/lib.rs

```diff
@@ -732,6 +753,47 @@ impl Batcher {
         true
     }
 
+    /// Handles a nonpaying message
+    /// this types of message are used in testing environments
+    /// here the batcher pays for verifying the proof for the user
+    async fn handle_nonpaying_message(
+        &self,
+        mut batch_state: tokio::sync::MutexGuard<'_, BatchState>,
+        nonced_verification_data: NoncedVerificationData,
+        ws_conn_sink: Arc<RwLock<SplitSink<WebSocketStream<TcpStream>, Message>>>,
+        nonce_value: U256,
+    ) -> bool {
```

Comment:
This function can never fail, but the return of the boolean is just to keep the consistency with the other methods and to account for a future failing point. Please tell me if I should better remove it.